File: src/main/java/me/kaotich00/fwauctionhouse/commands/api/Command.java
package me.kaotich00.fwauctionhouse.commands.api;

import org.bukkit.command.CommandSender;

public interface Command {

    void onCommand(CommandSender sender, String args[]);

    String getName();

    String getUsage();

    String getInfo();

    Integer getRequiredArgs();

}

File: src/main/java/me/kaotich00/fwauctionhouse/commands/api/Command.kt
package me.kaotich00.fwauctionhouse.commands.api

import org.bukkit.command.CommandSender

interface Command {

    val name: String

    val usage: String?

    val info: String?

    val requiredArgs: Int


    fun onCommand(sender: CommandSender, args: Array<String>)
}